examples/post_training_quantization/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,189 @@
+        print(f'Model weights (bin): {bin_size:.3f} Mb')
+        print(f'Model size:          {model_size:.3f} Mb')      
+    return model_size
+
+###############################################################################
+# Create a Tensorflow model and dataset
+
+def center_crop(image: tf.Tensor,
+                image_size: int,
+                crop_padding: int) -> tf.Tensor:
+    shape = tf.shape(image)
+    image_height = shape[0]
+    image_width = shape[1]
+
+    padded_center_crop_size = tf.cast(
+        ((image_size / (image_size + crop_padding)) *
+         tf.cast(tf.minimum(image_height, image_width), tf.float32)),
+        tf.int32)
+
+    offset_height = ((image_height - padded_center_crop_size) + 1) // 2
+    offset_width = ((image_width - padded_center_crop_size) + 1) // 2
+
+    image = tf.image.crop_to_bounding_box(
+        image,
+        offset_height=offset_height,
+        offset_width=offset_width,
+        target_height=padded_center_crop_size,
+        target_width=padded_center_crop_size)
+
+    image = tf.compat.v1.image.resize(
+        image,
+        [image_size, image_size],
+        method=tf.image.ResizeMethod.BILINEAR,
+        align_corners=False)
+
+    return image
+
+
+def preprocess_for_eval(image, label):
+    image = center_crop(image, 224, 32)
+    image = tf.keras.applications.mobilenet_v2.preprocess_input(image)
+    image = tf.image.convert_image_dtype(image, tf.float32)
+
+    label = tf.one_hot(label, DATASET_CLASSES)
+
+    return image, label
+
+
+val_dataset = tfds.load('imagenette/320px-v2', split='validation', 
+                        shuffle_files=False, as_supervised=True)
+val_dataset = val_dataset.map(preprocess_for_eval).batch(128)
+
+weights_path = data_utils.get_file('mobilenet_v2_imagenette_weights.h5', 
+                                   WEIGHTS_URL, cache_subdir='models')
+model = tf.keras.applications.MobileNetV2(weights=weights_path, 
+                                          classes=DATASET_CLASSES)
+
+###############################################################################
+# Quantize a Tensorflow model
+
+'''
+The transformation function transforms a data item into model input data.
+
+To validate the transform function use the following code:
+>> for data_item in val_loader:
+>>    model(transform_fn(data_item))
+'''
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+'''
+The calibration dataset is a small, no label, representative dataset
+(~100-500 samples) that is used to estimate the range, i.e. (min, max) of all
+floating point activation tensors in the model, to initialize the quantization
+parameters.
+
+The easiest way to define a calibration dataset is to use a training or
+validation dataset and a transformation function to remove labels from the data
+item and prepare model input data. The quantize method uses a small subset
+(default: 300 samples) of the calibration data set.
```

Comment:
```suggestion
The easiest way to define a calibration dataset is to use a training or
validation dataset and a transformation function to remove labels from the data
item and prepare model input data. Calibration dataset could be any type
but should be an iterable object and should yields pairs (input data, label).
The quantize method uses a small subset
(default: 300 samples) of the calibration data set.
```

Reply:
It's looks like this addition is wrong. Let's discuss it offline.